src/main/java/org/randywebb/wlts/beans/DetailedMinistered.java

```diff
@@ -14,127 +14,141 @@
-					ministered.put("nearestHouseholdName", household.nearest(households, relocation).get("householdName"));
-
-					results.add(ministered);
-
-				}
-
-			}
-
-		}
-
-		return results;
-	}
-
-	public DetailedMinistered() {
-	}
-
-	/**
-		@return A textual representation of the DetailedMinistered
-	*/
-	@Override
-	public String toString() {
-		return "DetailedMinistered [" + super.toString() + "]";
-	}
+    /** To use for debugging */
+    //private static Logger log = LoggerFactory.getLogger(DetailedMinistered.class);
+
+    /** Helper method to copy a field name from one object to another.
+        @param from The object to copy from
+        @param fromName The name of the key in from to copy into to
+        @param to The object to receive the value for key in from
+    */
+    private static void copy(AbstractBean from, String fromName, AbstractBean to) {
+        copy(from, fromName, to, fromName);
+    }
+
+    /** Helper method to copy a field name from one object to another with a different key.
+        @param from The object to copy from
+        @param fromName The name of the key in from to copy into to
+        @param to The object to receive the value for key in from
+        @param toName The name of the value in to
+    */
+    private static void copy(AbstractBean from, String fromName, AbstractBean to, String toName) {
+        to.put(toName, from.get(fromName));
+    }
+
+    /** Gets the CellProcessors for CSV generation.
+        @param headers The names of the fields, in the order they will be used.
+        @param stringProcessor The processor to use for the values
+        @return The appropriate processor for each header field
+                    in the order they are found in headers
+    */
+    public static CellProcessor[] csvProcessors(String[] headers, CellProcessor stringProcessor) {
+        ArrayList<String> none = new ArrayList<String>();
+
+        return csvProcessors(headers, none, none, none, stringProcessor, null, null, null);
+    }
+
+    /** Converts a List of District to a List of DetailedMinistered.
+        @param districts The list of ministering districts
+        @param households The households to look up names and addresses
+        @param relocation The data used to redirect the location for specified households
+        @return The DetailedMinistered List
+    */
+    public static List<DetailedMinistered> fromDistricts(List<District> districts,
+                                    List<Household> households, JSONObject relocation) {
+        ArrayList<DetailedMinistered> results = new ArrayList<DetailedMinistered>();
+        Map<String, Household> idToHousehold = Household.mapIndividualIdsToHousehold(households);
+
+        for (District district : districts) {
+            Household districtHousehold = null == district.getDistrictLeaderIndividualId()
+                        ? null
+                        : idToHousehold.get(district.getDistrictLeaderIndividualId());
```

Comment:
These short line splits are killing me...

Reply:
When I increase the line limit to 200, I'll fix these